api/src/middlewares/services/utils/__tests__/getFormType.test.ts

```diff
@@ -0,0 +1,24 @@
+import { getFormType } from "../getFormType";
+
+test("Returns the correct form type from the service field if the service field is available", () => {
```

Comment:
The test name should include the function you're testing for readability. 

Otherwise you can wrap it in a `describe` or `suite`, but imo most of the time it's unnecessary to do so